### DISCUSSION TITLE
Dummy question about Firefly III Importer

### DISCUSSION THREAD

**Discussion by boelenz** [firefly-iii/firefly-iii#6374] [Firefly III]
Hi,

I am completely new to Firefly. A good friend has helped me with a Docker-Compose based setup of Firefly III and this is working. My Firefly III implementation is hosted somewhere where I can access it through my web browser. Now, I would like to import my bank data (Bunq, ING, ABN-AMRO) into Firefly. I have read the documentation of Firefly Importer, but to this point, I have not been able to do any imports. 

Please show me the basic steps on how to do import; I do not know how to set it up.

Can I use a client computer and the CLI importer with the API key of my Firefly implementation? The bank data would then be posted using REST to my Firefly implementation? That would be understandable for me, but I see no explanation of this setup in the documentation.

Thanks a lot for anyone that can help me proceed.

**Comment by JC5** [comment]
How dummy you'll need the explanation to be depends a bit on your own tech-level. Since there is no unified API between those banks, you have to use either:

- Separately downloaded CSV files and an accompanying config file ([needs some trial and error](https://github.com/firefly-iii/import-configurations/tree/main/nl))
- A connection (3x) with Salt Edge to each bank.

It's easiest to set it up using Docker in the UI the first time:

```
docker run \
--rm \
-e FIREFLY_III_ACCESS_TOKEN=(here) \
-e FIREFLY_III_URL=(here) \
-p 8081:8080 \
fireflyiii/data-importer:latest
```

Browse to localhost:8081 and connect to your Firefly III instance following the instructions. Then, continue with CSV files or Salt Edge. After the first setup you can download your own config file and use it.
